src/main/kotlin/app/revanced/patches/youtube/misc/zoomhaptics/annotations/ZoomHapticsCompatibility.kt

```diff
@@ -0,0 +1,13 @@
+package app.revanced.patches.youtube.misc.zoomhaptics.annotations
+
+import app.revanced.patcher.annotation.Compatibility
+import app.revanced.patcher.annotation.Package
+
+@Compatibility(
+    [Package(
+        "com.google.android.youtube", arrayOf("17.43.36")
```

Comment:
I believe it is fine to remove the versions array and make it compatible with all future versions. Other than that this should support the targeted, latest version as well.

Reply:
Done, removed versions array